src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -363,38 +363,41 @@ void ConnectionHandlerImpl::OnSessionStartedCallback(
     const BsonObject* params) {
   LOG4CXX_AUTO_TRACE(logger_);
 
-  uint32_t new_session_id = 0;
-  uint32_t hash_id = protocol_handler::HASH_ID_WRONG;
+  std::vector<std::string> empty;
```

Comment:
@AKalinich-Luxoft please specify what is in this vector.

Reply:
@LuxoftAKutsan renamed in https://github.com/smartdevicelink/sdl_core/pull/1971/commits/a87c65251e81cb914deaa1f0fe1716175fe1270d